etna/packages/etna-js/components/Nav.css

```diff
@@ -0,0 +1,88 @@
+.etna-nav {
```

Comment:
This isn't really a nit but a cool thing to note: the way the css loader works is that it actually name mangles css classes to prevent collisions.  When you do the import with require, it returns a hash that maps the local class names to their true class names for purpose of controlling them in React.

A side effect is that you can totes use generic and less specialized names without fear of collision.  `nav` or even `component` is sufficient.  But you don't need to go through the effort of changing it, just noting.

Reply:
https://webpack.js.org/loaders/css-loader/

Reply:
Specifically, check the scoping features under Features